packages/css-framework/src/components/_phase-banner.scss

```diff
@@ -0,0 +1,25 @@
+.rn-phase-banner {
+  display: block;
+  background-color: color(primary, 100);
+  padding: spacing(2) spacing(6);
```

Comment:
Do we want to place the content of the banner inside a `.container` class? This way it'll scale and stay horizontally in line with the rest of the content

Reply:
Can you explain what you mean? Where should the `.container` class be? Does it require an extra DOM element and can you point me to the file I should be looking it which describes what the `container` class does so I know when to use it in the future?

Reply:
The container class sits around the content, so in this case it would be:
```
<div className="rn-phase-banner">
    <div class="container">
        content here
    </div>
</div>
```

There should be a site-wide container element. If it doesn't exist, it needs to be created. It will provide horizontal padding for all the elements in the site, and will assert a max width on the content if the browser window is opened to a larger size.